ci_framework/roles/cifmw_cephadm/tasks/selinux.yml

```diff
@@ -18,7 +18,7 @@
   block:
     - name: Set fcontext on ceph config and admin keyring
       become: true
-      sefcontext:
+      ansible.builtin.sefcontext:
```

Comment:
that's `community.general.sefcontext`:
https://docs.ansible.com/ansible/latest/collections/community/general/sefcontext_module.html